specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md

```diff
@@ -0,0 +1,115 @@
+# HDInsight On Aks
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for HDInsight On Aks.
+
+---
+
+## Getting Started
+
+To build the SDK for HDInsight, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the HDInsight On Aks API.
+
+``` yaml
+title: HDInsightOnAksManagementClient
+description: HDInsight On Aks Management Client
+openapi-type: arm
+openapi-subtype: rpaas
+azure-arm: true
+tag: package-2023-06-preview
+```
+
+### Suppression
+
+
+### Tag: package-2023-06-preview
+These settings apply only when `--tag=package-2023-06-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-06-preview'
+input-file:
+  - preview/2023-06-01-preview/hdinsight.json
+suppressions:
+  - code: MISSING_APIS_IN_DEFAULT_TAG
```

Comment:
Same remark as [here](https://github.com/Azure/azure-rest-api-specs/pull/24940/files#r1282607706)

Reply:
removed, thanks @konrad-jamrozik  for your comment.